package.json

```diff
@@ -31,6 +31,7 @@
     "start-with-fixtures": "node utils/sails_with_fixtures.js",
     "debug": "node debug app.js",
     "test": "./node_modules/.bin/istanbul cover node_modules/mocha/bin/_mocha",
+    "postinstall": "./node_modules/.bin/precommit install",
```

Comment:
By the way, there is no need to specify the full path to the binary, npm will look in `./bin` for the executable, so `"postinstall": ".precommit install",` will do the job.